lib/index.js

```diff
@@ -119,9 +147,9 @@ function StickyInlines(opts) {
 
   return {
     onKeyDown,
-    onChange,
-    onSelect,
-  }
+    onEditor,
```

Comment:
`onEditor` doesn't exist anywhere in this fie

Reply:
Good catch @tommoor!